account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -503,9 +505,10 @@ def find_repartition_by_templates(
                     )
                 )
 
-        # Mark to be removed the lines not found
-        remove_ids = [x for x in current_repartition.ids if x not in existing_ids]
-        result += [(2, x) for x in remove_ids]
+        if tax.amount_type != "group":
```

Comment:
We should prevent the creation of such default lines instead of masking this change, isn't it?

Reply:
What do you mean by "masking this change"?

Reply:
ping @pedrobaeza

Reply:
On tax of type `group`, we mask any problem/update. The solution is to prevent the creation of such default lines.

Reply:
But the creation of those reparation lines is due to the default value set in Odoo standard, do you think Odoo would accept the change (specially in 14.0)?

Reply:
I think there are several ways of doing them from here:

- Inhibit them through context and override.
- Delete them later.
- Be creative!

Reply:
OK, let's go with this.